support-frontend/assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -964,6 +939,66 @@ form {
   margin-top: $gu-v-spacing;
 }
 
+.gu-content--frontline-campaign {
+  @include mq(desktop) {
+    .gu-content__main::before {
+      content: '';
+      height: 414px;
+      width: 100%;
+      position: absolute;
+      left: 0;
+      top: 0;
+      background: no-repeat url('https://media.guim.co.uk/b9ed1767866d29186f198b523cd5571aeda02a88/0_1269_5184_1516/2000.jpg');
```

Comment:
Would be interested in @paperboyo's opinion on this. We can't use `srcset` to serve a lower-res image on smaller screens, because we're using a background image in CSS. But given that there's no background image until desktop, is there much to be gained from serving different images between desktop/leftCol/wide? Not sure what current best practice is. (The image is 220K)

Reply:
Not sure what's the latest on this, but [here's the horse's mouth](https://stackoverflow.com/questions/26801745/is-there-a-srcset-equivalent-for-css-background-image).

Irregardless, even for just one asset, if it's to be simply a static one, Grid crops aren't terribly well compressed and are sized at eg. 1000px and 2000px, when the asset's dimensions should be close to how it appears at dpr=1 (or more if we would want to satisfy at least some HiDPI). 220KB seems excessive, if it's just a one off, I would compress in squoosh and upload via s3 uploader instead.

Reply:
got it from 220KB to 103KB with squoosh! great shout